Robot_Adapter/CRUD/Read/Properties/Material.cs

```diff
@@ -133,6 +105,22 @@ private void ReadMaterialNamesFromDB(string dbName)
 
         /***************************************************/
 
+        private IMaterialFragment ReadMaterialFromPanel(IRobotObjObject robotPanel)
+        {
+            IRobotLabelServer robotLabelServer = m_RobotApplication.Project.Structure.Labels;
+            IRobotLabel thicknessLabel = robotPanel.GetLabel(IRobotLabelType.I_LT_PANEL_THICKNESS);
+            IRobotThicknessData thicknessData = thicknessLabel.Data;
+            IMaterialFragment material = null;
+            try
+            {
+                material = MaterialFromLabel(robotLabelServer.Get(IRobotLabelType.I_LT_MATERIAL, thicknessData.MaterialName));
+            }
+            catch
+            {
+                BH.Engine.Reflection.Compute.RecordWarning("No material is assigned for panel " + robotPanel.Number.ToString());
```

Comment:
This error implies that a material should be assigned and that is only affects the named panel. 
The warning appears for loading panels, which I'm unsure if they should have a material (they can have it).
But also only names the first panel and then processed to not assign material for the rest of them as well.

Reply:
Thanks! Sorry, this one was intended to be a quick fix, turns out there were quite a few issues with the panel property code. Good to make this as robust as possible. I've just removed the material read if it's a panel and tested - works ok, however, I want to just add another panel type (not meshed) that _is_ able to have a material, and make sure that's not treated as 'cladding'.

Reply:
All good. Test files updated, and checked meshed and unmeshed. 

![image](https://user-images.githubusercontent.com/15233608/73547103-36691300-4436-11ea-95ce-5f1ccab47b00.png)